device-types/Digi/Digi-Passport-32.yml

```diff
@@ -0,0 +1,84 @@
+---
+manufacturer: Digi
+model: Digi Passport 32
+slug: digi-passport-32
+part_number: '70002262'
+u_height: 1
+is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (6.23 lbs or 2.85 kg)